<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UdpWidgetClass</class>
 <widget class="QWidget" name="UdpWidgetClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>439</width>
    <height>312</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>UdpWidget</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="1" colspan="2">
    <widget class="QLineEdit" name="ipEdit"/>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>IP</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="8">
    <widget class="QTextEdit" name="dataEdit"/>
   </item>
   <item row="3" column="0" colspan="8">
    <layout class="QHBoxLayout" name="sendLayout"/>
   </item>
   <item row="0" column="3">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>PORT </string>
     </property>
    </widget>
   </item>
   <item row="0" column="5" colspan="2">
    <widget class="QLineEdit" name="portEdit"/>
   </item>
   <item row="0" column="7">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
